.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
does this block me from ignoring a template and just writing my own issue? I would like to keep the option to ignore a template.

Reply:
Yes, it does. However, you'd still be free to delete the template text in the submission.